product_docs/docs/pgd/5/cli/command_ref/pgd_switchover.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: switchover
+---
+
+Switchover to new write leader.
```

Comment:
For parallelism with other topics, would it be correct to make this two words:

Switches over to new write leader.